### DISCUSSION TITLE
We're looking for new name + logo ideas

### DISCUSSION THREAD

**Discussion by throwaway-d** [spikecodes/libreddit#709] [Ideas]
> Come up with a new name for Libreddit. We have to do this, unfortunately, because of the [probable Reddit API TOS violation](https://github.com/libreddit/libreddit/issues/585#issuecomment-1251685606). This is where I'd like community input. Admittedly, I'm biased toward "Ferrit" simply because I think the name is cute, which is partly why I think we should have the public decide on this, precisely because of my bias.

https://github.com/libreddit/libreddit/issues/599#issuecomment-1299534520

**Comment by spikecodes** [comment]
As [discussed by Daniel](https://github.com/libreddit/libreddit/issues/585#issuecomment-1251685606), Libreddit's name may be in violation of Reddit's TOS as it technically uses "reddit" in the name. To future-proof this project against any alleged copyright infringement, we should come up with a new name (and thus, a new logo).

We're looking for suggestions so feel free to submit your ideas here!

**Reply by throwaway-d** [reply]
I also love `Ferrit` name.

**Reply by cacfd3a** [reply]
I like Leddit too, as a contraction of the current name. Could be read as 'light client for reddit', phonetically close and humorous callback to 4chan snark.

**Comment by spikecodes** [comment]
The new name should be unique, so as to be distinguishable in search results from other sites and projects, but also short so it's easy to type and to tell others the name. This could be achieved by removing the `e` from a word ending in `er`, which is a common naming convention for tech products.

Some name ideas I had:

- _Frontr_
    - a double entendre for "**front** page of the internet" (reddit's slogan) and "private **front**-end to Reddit" (libreddit's slogan)
    - could possibly pronounced "frontier"
- _Puffr_
    - **P**rivate
    **U**
    **F**ront-end
    **F**or
    **R**eddit
    - We could use a puffer fish as the mascot, but with a more modern icon than OpenBSD's

**Reply by throwaway-d** [reply]
> could possibly pronounced "frontier"

Sounds like a shooting video game. And I don’t like that sound.

**Reply by throwaway-d** [reply]
> We could use a puffer fish as the mascot, but with a more modern icon than OpenBSD's

Maybe that would be too easy to be confused with OpenBSD. _An alien pufferfish_ would be better.

**Reply by spikecodes** [reply]
I like the alien pufferfish idea!

**Reply by Daniel-Valentine** [reply]
I also thought of OpenBSD when I saw _Puffr_. That would be a great name for a product that complemented or which was derived from OpenBSD in some way, but since this isn't, I'm not enthusiastic about it.

**Reply by Daniel-Valentine** [reply]
I also gave the "-Cr" (where C is a constant) idea some thought and I think we should avoid it. It can look cute but it can also look like a misspelling.

**Reply by spikecodes** [reply]
Yeah, that makes sense.

**Comment by Daniel-Valentine** [comment]
We could always revisit _Ferrit_.

A couple of other ideas I've considered:

* ~~_Rettr_~~ (rejected: [see my reasoning here](https://github.com/libreddit/libreddit/discussions/709#discussioncomment-4951562))
* ~~_Russit_~~ (rejected: potential trademark clash with [company providing services in a similar space](https://www.linkedin.com/company/russit/))
* _Rudder_ (a little silly but I kind of like it)
* ~~_Reviewer_~~ (rejected: too generic)
* _Ruffit_
* _Librebacon_
* _Riti_ (from Maori _rīti_ "to read")

Of these names, I personally prefer _Ruffit_.

**Reply by throwaway-d** [reply]
“Reviewer” is too generic.

**Reply by throwaway-d** [reply]
There is already a company named "Russit" https://www.linkedin.com/company/russit (I wish there is a front-end for LinkedIn)
And some shoes, too https://www.amazon.com/DV-Dolce-Vita-Womens-Russit/dp/B08XSW4SDX

**Reply by throwaway-d** [reply]
"Librebacon"
Odd name. Would be more suitable for a restaurant like [Libre Foods](https://www.librefoods.co)

**Reply by Daniel-Valentine** [reply]
> "Librebacon" Odd name. Would be more suitable for a restaurant like [Libre Foods](https://www.librefoods.co)

I originally gave the Ferrit fork the name _Libbacon_ in reference to the Reddit shibboleth ["When does the narwhal bacon?"](https://knowyourmeme.com/memes/the-narwhal-bacons-at-midnight), before deciding that it made the program sound more like a library than it would a stand-alone binary. _Librebacon_ would address the "lib-" issue while also still paying homage to that shibboleth.

**Reply by Daniel-Valentine** [reply]
> There is already a company named "Russit" https://www.linkedin.com/company/russit (I wish there is a front-end for LinkedIn) And some shoes, too https://www.amazon.com/DV-Dolce-Vita-Womens-Russit/dp/B08XSW4SDX

It looks like they specialize in web/app development. That's too close for comfort and we would be infringing on their trademark. So I agree that _Russit_ is a no-go.

**Reply by spikecodes** [reply]
I like Rudder and Riti but I think the Librebacon reference is too specific for people to get and the name is a bit long. This is purely subjective but Ruffit just doesn't stick with me.

It would be nice to be able to attach some meaning to Rudder but I can't think of any.

**Reply by Daniel-Valentine** [reply]
> It would be nice to be able to attach some meaning to Rudder but I can't think of any.

***R***udder, ***U***nambiguously, ***D***raws ***D***ata, ***E***xpetidiously, from ***R***eddit?

**Reply by domve** [reply]
I do think Ferrit would still be the most fitting if a name change were to occur.

Ruffit and Rudder seem too generic and ambigous in my opinion, compared to other names in the space like Teddit and Nitter, it's not entirely clear what the project is for. Searching Ruffit also apparently turns up with [a dog carrier](https://www.amazon.com/Ruffit-Dog-Carrier-Backpack-Medium/dp/B078QWCHRL).

**Reply by Daniel-Valentine** [reply]
Thanks, everyone, for your input.

Based on the above, looks like _Riti_ is probably the best of the options I've offered, if not the least offensive. What do we think?

**Reply by throwaway-d** [reply]
![image](https://user-images.githubusercontent.com/121148372/223421053-c0296e37-c03c-436a-947f-f790415aabb3.png)
![image](https://user-images.githubusercontent.com/121148372/223421092-e116e832-a1c8-49bf-98fa-c747e8d3af30.png)

**Reply by throwaway-d** [reply]
Now just like what @binaryben did below, I think you should make concept arts for the logo name with icon using the names above. Then we can choose what’s the best.

**Comment by sigaloid** [comment]
Honestly I'm okay with any of these suggestions. I've been trying to come up with something better but have come up empty. Wish I had more to share :(

**Comment by binaryben** [comment]
I like Ferrit. Can subtly alter Snoo to look more like a ferret and use a different colour for a logo. That should be enough to fend off lawyers.

**Reply by throwaway-d** [reply]
Wow! Ferret mascot is a brilliant idea! I think we can have a rusty iron ferret?

**Reply by binaryben** [reply]
Not really sure what is meant by an iron ferret... here is what I had in mind though:

![ferrit logo concept](https://user-images.githubusercontent.com/12827556/231454733-6d3c4e97-d019-4482-9018-3f9ecad49276.png)

**Reply by throwaway-d** [reply]
That looks nice but still feel too similar to Reddit. Maybe we have to remove the _antenna_ and change the font

**Reply by throwaway-d** [reply]
> remove the antenna

Or make the ferret have **2** antennas?

**Reply by binaryben** [reply]
IANAL, but that should pass the, "*close but not quite reddit*" test from a legal perspective. Interestingly, the font isn't the same as used in the Reddit logo. It's a free (💪) font called Rocko FLF that is similar, but again slightly off in key ways - I feel a nice nod to this being a FOSS project. 

I am not willing to change the font or remove the antenna myself as my goal was to show it is derived from Reddit but not affiliated or endorsed by Reddit. Happy to provide my Affinity Designer file (the artwork is licensed under CC 4.0 SA-BY as far as I am concerned for the moment) for others to alter though.

***

EDIT

Got curious, went to look for [the official word on their trademarks](https://www.redditinc.com/policies/trademark-use-policy). This is clearly not a Snoo (Foo here has a nose and very different proportions). Does not use the wordmark or trademark colour. And there is no mention of the antenna itself being trademarked. Should be all clear 👍

**Comment by cameronj86** [comment]
Ferrit has my vote  ✊🏾

**Comment by jobukkit** [comment]
"rproxy" is what I used as the subdomain, not an amazing name but you could use that too.

**Comment by Opening-Button-8988** [comment]
Libresnoo.

Not my best work but it's honest.

**Reply by sigaloid** [reply]
Snoo has the same trademark :(

**Comment by ixfd64** [comment]
I have a few ideas:

* tidd(e)r
* derrit
* libalien
* libspezsucks

**Comment by Opening-Button-8988** [comment]
Personally, and I know it's a bit trite, but I would like to see anything with lib or libre in the name. I know it's overused but hey, it might draw people away from shitty proprietary for-profit organizations like reddit and give more attention to FOSS, linux, and decentralization in general.

**Reply by Tokarak** [reply]
It's also more descriptive than any animal mascot.

**Reply by binaryben** [reply]
Interestingly, I am the opposite. It is so boring and overused. I also suspect it'll actually have the opposite effect with the majority seeing anything with "lib" in the name as being nerdy/geeky/too hard.

**Comment by MarcoBrandao** [comment]
My take on a new name is Libideias or Libthoughts. Maybe Libposts but the is a project at Github called Libpostal.

**Comment by MichalSvatos** [comment]
`Ferrit` is not bad. 
What about `teddir`. And as an icon some kind of bear :thinking: 

EDIT: 
<img width="343" alt="image" src="https://github.com/libreddit/libreddit/assets/5910531/caca8057-67e0-45c0-bc85-bd89cf2e32fc">
(quick test, icon is a modified wolf from iconoir.com library)

**Reply by axeII** [reply]
Both sounds cool. 👍🏻

**Reply by throwaway-d** [reply]
That is too similar to Teddit. https://codeberg.org/teddit/teddit

**Reply by dikiaap** [reply]
Libursa (libre+[ursa](https://en.wikipedia.org/wiki/Ursa))?

**Comment by jmcelroy01** [comment]
So an earlier idea I had was something like "freereddical" in an attempt to keep something like "lib" in the term, but those contribute to oxidative stress, it sounds awkward, and it's probably better to avoid a term with political connotations. So how about what counters free radicals? Drop the anti- part, and you get -

Oxiddet. 

Unsure whether 2 or 1 d's in it looks better. 

It has the perk of suggesting the simplified frontend is a breath of fresh air, and a search didn't turn up any usage of the term. 

edit: oxydet? that's a product name, so one could add an i in to make it oxiydet.

**Comment by Cwpute** [comment]
Some suggestioms have been made in issue #822 , i'll miror some of mine and another one here just i case.

What about `reddirect` ? as suggested by @Opening-Button-8988
It keeps almost 90% of the original brand's name *reddi-* that associates it with reddit while being obvious about what it actually does, that is "redirecting" the data from reddit to an opentource front-end.

**Comment by Cwpute** [comment]
`reddzat`, as in "i read that yesterday"
sounds like a direct response to the original brand: read it, read that… it keeps the recognizable double D, part of the original name and its sens and tone.
Also: same number of syllables, same typed lenght, short and efficient, while staying on its own legs as it sounds different and looks different.

Might be the most efficient alternative name i think.

**Comment by Cwpute** [comment]
I thought maybe playing with the word "ditto" could give us interesting results…

I'd suggest `ReDitto`: it sounds and look like the original name, with a significant and recognizable *-o* in the end that make it sound different quite easily.
Although i see two cons: it has a double T instead of a double D, which can throw off some users, and the joke in its name is doubled, as the prefix *re-* and the word *ditto* essentialy mean the same thing: a recreation, a double, a second one.

Other ways to use "ditto" might include: dditto, Redito, rredditto, Riditto, etc…

**Comment by Cwpute** [comment]
`Rabbit` 🐰

**Comment by Cwpute** [comment]
`Ribbit` 🐸

**Comment by Tokarak** [comment]
`Rabbid` 🐶
